packages/vue-to-dsl/src/converter.ts

```
@@ -0,0 +1,1100 @@
+      if (!fileMap.has(baseName)) {
+        fileMap.set(baseName, [])
+      }
+      fileMap.get(baseName)!.push(relativePath)
+    }
+
+    // Determine which files need special naming (camelCase with directory prefix)
+    const needsSpecialNaming = new Set<string>()
+    for (const paths of fileMap.values()) {
+      if (paths.length > 1) {
+        // Multiple files with same basename, all need special naming
+        paths.forEach((p) => needsSpecialNaming.add(p))
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)